src/test/java/it/gov/pagopa/arc/service/bizevents/BizEventsServiceImplTest.java

```diff
@@ -47,19 +50,20 @@ class BizEventsServiceImplTest {
     private static final String FILTER = "DUMMY_FILTER";
     private static final String TRANSACTION_ID = "TRANSACTION_ID";
     private static final String DUMMY_FISCAL_CODE = "FISCAL-CODE789456";
+    private static final String CONTINUATION_TOKEN = "continuation-token";
 
-    @Autowired
     private BizEventsService bizEventsService;
 
     @Mock
-    private BizEventsConnector bizEventsConnectorMock;
+    private static BizEventsConnector bizEventsConnectorMock;
     @Mock
-    private BizEventsTransactionDTO2TransactionDTOMapper transactionDTOMapperMock;
+    private static BizEventsTransactionDTO2TransactionDTOMapper transactionDTOMapperMock;
     @Mock
-    private BizEventsTransactionsListDTO2TransactionsListDTOMapper transactionsListDTOMapperMock;
+    private static BizEventsTransactionsListDTO2TransactionsListDTOMapper transactionsListDTOMapperMock;
```

Comment:
I am not sure that this variables should be static... as also the @Mock defined below